Lib/test/test_uuid.py

```diff
@@ -674,26 +674,57 @@ class TestUUIDWithExtModule(BaseTestUUID, unittest.TestCase):
 class BaseTestInternals:
     _uuid = py_uuid
 
+    # data is specific to AIX - with '.' as _MAC_DELIM
+    # and strings shorter than 17 bytes (no leading 0)
+    @unittest.skipUnless(_AIX, 'requires AIX')
```

Comment:
Similar to the skip logic changes that you extracted out to other PRs, a better check here will be `@unittest.skipUnless(self.uuid._MAC_DELIM == b'.')`, since that's what the sample data is assuming.

Reply:
That also suggests abstracting out a marker attribute for `uuid._MAC_OMITS_LEADING_ZEROES`, so the full check would be:

    @unittest.skipUnless(self.uuid._MAC_OMITS_LEADINGS_ZEROES)
    @unittest.skipUnless(self.uuid._MAC_DELIM == b'.')

Reply:
You (or I) resolved this. Both marker attributes are defined, and rather than skip, are assigned to the mock tests.

If I have missed something - I'll get on it, if not - please consider approving the changes, Thx.

Reply:
The issue is that this is a valid test anywhere that those two conditions are true, so rather than testing for "AIX_" here, you can just test for the MAC notation conventions that the test cares about.